siteapp/views.py

```diff
@@ -1501,13 +1559,16 @@ def edit_portfolio(request, pk):
             form = PortfolioForm(request.POST, instance=portfolio)
             if form.is_valid():
                 form.save()
+            # Log portfolio update
+            messages.add_message(request, messages.INFO, f"The portfolio '{portfolio.title}' has been updated.")
         except IntegrityError:
             messages.add_message(request, messages.ERROR, "There is different Portfolio with this name.")
         return redirect("list_portfolios")
 
     return render(request, 'portfolios/edit_form.html', {
         'form': form,
-        'portfolio': portfolio.title,
+        # 'portfolio': portfolio.title,
+        'portfolio': portfolio,
         "can_change_portfolio": request.user.has_perm('change_portfolio', portfolio),
```

Comment:
Since this condition includes the "change_portfolio" permission but also the "is_superuser" condition, maybe we should just pass in the "CAN_EDIT_PORTFOLIO" boolean from above, since it's already been calculated above. Avoids duplicating the check for the user permission.